src/common/tests/pages/login.spec.tsx

```diff
@@ -10,14 +10,14 @@ describe('Pages/Login test', () => {
     expect(wrap).toMatchSnapshot();
```

Comment:
you have to mount it to match a snapshots

Reply:
fix here too eb52e9cf7ef6a4c6d91c185facf56653e8c3bc4b